minio_storage/storage.py

```diff
@@ -39,7 +38,7 @@ def __init__(
         bucket_name: str,
         *,
         base_url: T.Optional[str] = None,
-        file_class: T.Optional[T.Type[File]] = None,
+        file_class: T.Optional[T.Type[MinioStorageFile]] = None,
```

Comment:
This was actually the wrong class before. Django's `File` doesn't take a 3rd constructor argument, whereas the subclass `MinioStorageFile` does. We call `self.file_class(...)` with 3 arguments.

This doesn't affect any default behavior, it only clarifies what kinds of non-default `file_class` a user may override.